docs: add api collection and update readme for data exchange tutorial (#155)
eclipse-tractusx/tractus-x-umbrella: docs/README.md

# Introduction

> This repository provides the API Documentation for the Umbrella.

Reviewer: ```suggestion
This repository provides the API requests to provide a tutorial for the Data exchange.
```

> This repository provides the API requests to provide a tutorial for the Data exchange.

Reviewer: ```suggestion
This repository provides the API requests to provide a tutorial for the Data Exchange.
```

The following table links you to the respective documentations.

| Documentation                                                    | Purpose                                                                                 |
|------------------------------------------------------------------|-----------------------------------------------------------------------------------------|

> | [API Documentation](api/API_Doc.md)                              | Information about the endpoints.                                                        |

Reviewer: ```suggestion
| [API Requests](api/API_Doc.md)                              | API Requests for Data Exchange.                                                        |
```

## NOTICE

This work is licensed under the [Apache-2.0](https://www.apache.org/licenses/LICENSE-2.0).

- SPDX-License-Identifier: Apache-2.0
- SPDX-FileCopyrightText: 2024 Contributors to the Eclipse Foundation

> - Source URL: <https://github.com/eclipse-tractusx/ssi-authority-schema-registry>

Reviewer: ```suggestion
- Source URL: <https://github.com/eclipse-tractusx/tractus-x-umbrella>
```
